server/main.go

```diff
@@ -29,12 +27,21 @@ func (s *server) GetUsers(ctx context.Context, r *pb.GetUsersRequest) (*pb.GetUs
 	return &pb.GetUsersResponse{Users: users}, err
 }
 
+// CreateUser
+func (s *server) CreateUser(ctx context.Context, r *pb.CreateUserRequest) (*pb.CreateUserResponse, error) {
+	var _, err = models.CreateUser(*r)
+	if err == nil {
+		return &pb.CreateUserResponse{}, nil
+	} else {
+		return &pb.CreateUserResponse{}, err
+	}
```

Comment:
Goではこんな感じで実装するのがお約束になっています。
なにも考えずにこんな感じで実装してしまってください！

```suggestion
	if err != nil {
		return nil, err
	}
	return &pb.CreateUserResponse{}, nil
```